### DISCUSSION TITLE
Invite Users/Share buttons

### DISCUSSION THREAD

**Discussion by Ievgenii78** [apache/superset#25027] [Q&A / Help] [answered]
Currently, I'm working on 2.1 branch, after installing I do not see Invite user and also share as its shown in screenshot below:
![Screenshot at 17-13-09](https://github.com/apache/superset/assets/76782903/f40acb04-4069-4aa2-95e6-5ac1ab2a03f8)

Mine looks like this:
![Screenshot at 17-14-20](https://github.com/apache/superset/assets/76782903/42e6149a-5c0d-42f1-a625-aed3b8430121)

**Comment by michael-s-molina** [comment, marked as answer]
Hi @Ievgenii78. This is a Preset only feature and it's not available in standard Superset.

**Reply by Ievgenii78** [reply]
@michael-s-molina  Got it, Thanks for info.
